react/table/.env

```diff
@@ -0,0 +1,6 @@
+REACT_APP_DOMAIN=http://localhost:8081/services/jhipster/api
```

Comment:
given that the .env file won't be provided in production, I'd explain how to create .env | .env.local in the readme (something like https://github.com/entando/entando-cms/blob/master/README.md#configuration-example)

Reply:
added